config/environments/demo.rb

```diff
@@ -6,4 +6,6 @@
   config.action_mailer.default_options = { from: 'hello@mg-demo.getyourrefund-testing.org' }
   config.action_mailer.default_url_options = { host: 'demo.getyourrefund.org' }
   Rails.application.default_url_options = config.action_mailer.default_url_options
+  # Custom config value read by DeviseMailer class
+  config.devise_email_from = 'no-reply@mg-demo.getyourrefund-testing.org'
```

Comment:
Would be nice to explain, what this email is for (so that we can implicitly understand why it is different) and add that comment all environment config files.

Reply:
`config.address_for_transactional_authentication_emails = ...`